lib/vsc/install/commontest.py

```diff
@@ -95,7 +102,6 @@ class CommonTest(TestCase):
         'undefined',
         'no-value-for-parameter',
         'dangerous-default-value',
-        'redefined-builtin',
```

Comment:
why is this removed? isn't adding the blacklist entry sufficient?

Reply:
I think it is being removed in favor of the blacklist entry, which is more specific for the intended bullitin:

```
"Redefining built-in 'basestring'"
```

Reply:
i think this disables all redefined builtin errors. you should keep this one